actions/Import.php

```diff
@@ -23,9 +23,11 @@
 
 use common_report_Report;
 use core_kernel_classes_Resource;
+use oat\generis\Helper\UserHashForEncryption;
```

Comment:
Not found

Reply:
PR created.https://github.com/oat-sa/generis/pull/480